### PR TITLE
Adapt frontend app to the same style as other aragon apps

### PR DESCRIPTION
**Pull request by bingen**
Use babel.
Use SafeMath for contract.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
 build
 .cache
-#dist
\ No newline at end of file
+dist
+package-lock.json
```
[the remark this answers is not in the document]
Comment (bingen) [reply .gitignore]: Besides, it seems that it's used for IPFS too, so I should remove then `dist` too: https://github.com/aragon/aragon-cli/blob/master/src/commands/apm_cmds/publish.js#L102
I'll try to think of a better solution.

File: app/script.js [hunk 2]
```diff
@@ -5,15 +5,26 @@ const app = new Aragon()
...
   if (state === null) state = initialState
 
   switch (event.event) {
     case 'Increment':
-      return { count: parseInt(state.count, 10) + parseInt(event.returnValues.step, 10) }
+      return { count: await getValue() }
     case 'Decrement':
-      return { count: parseInt(state.count, 10) - parseInt(event.returnValues.step, 10) }
+      return { count: await getValue() }
     default:
```
Comment (bingen) [inline app/script.js]: I guess you could get rid of the switch, or at least join those 2 cases.

File: app/script.js [hunk 3]
```diff
@@ -5,15 +5,26 @@ const app = new Aragon()
...
+app.store(async (state, event) => {
   if (state === null) state = initialState
 
   switch (event.event) {
     case 'Increment':
-      return { count: parseInt(state.count, 10) + parseInt(event.returnValues.step, 10) }
+      return { count: await getValue() }
     case 'Decrement':
-      return { count: parseInt(state.count, 10) - parseInt(event.returnValues.step, 10) }
+      return { count: await getValue() }
```
Comment (bingen) [inline app/script.js]: I guess you could get rid of the switch, or at least join those 2 cases.

Comment (sohkai) [reply app/script.js]: Wanted to keep the switch so it was obvious what the pattern would be for a more complicated app :).

**Pull request merged**